Nrf cc3xx 0.9.22 (#2146)
nrfconnect/sdk-nrfxlib: crypto/doc/CHANGELOG_cc3xx_platform.rst

All notable changes to this project are documented in this file.

nrf_cc3xx_platform - 0.9.22
***************************

New version of the library with the following features:

>  * Restructured the code for `CTR_DRBG`` and `HMAC_DRBG`` to avoid dependency on both (which was causing size increase in v0.9.21).

Reviewer: ```suggestion
 * Restructured the code for `CTR_DRBG`` and `HMAC_DRBG`` to avoid dependency on both (which would cause a size increase in v0.9.21).
```

Reviewer 2: Edited Tomi's suggestion as mine would target the same word.

>  * Restructured the code for `CTR_DRBG`` and `HMAC_DRBG`` to avoid dependency on both (which would cause a size increase in v0.9.21).

Reviewer 2: ```suggestion
 * Restructured the code for ``CTR_DRBG`` and ``HMAC_DRBG`` to avoid dependency on both (which would cause a size increase in v0.9.21).
```

>  * Removed usage of legacy Mbed TLS APIs previously used for KMU support.

Reviewer 2: ```suggestion
 * Removed the usage of legacy Mbed TLS APIs that were previously used for KMU support.
```
Only of those used for KMU or all legacy ones?

>  * Updated the library to build with MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG enabled

Reviewer 2: ```suggestion
 * Updated the library to build with ``MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG`` enabled.
```

> Library built against Oberon PSA crypto v2.0.0.

Reviewer 2: ```suggestion
Library built against Oberon PSA Crypto v2.0.0.
```
